crates/bevy_render/src/render_resource/shader.rs

```diff
@@ -21,6 +21,17 @@ pub enum ShaderReflectError {
     #[error(transparent)]
     Validation(#[from] naga::WithSpan<naga::valid::ValidationError>),
 }
+
+/// Describes whether or not to perform runtime checks on shaders.
+/// Runtime checks can be enabled for safety at the cost of speed.
+/// By default no runtime checks will be performed.
```

Comment:
can we please add a note that no checks are performed for spirv, regardless of this setting?

... or if we panic for validated spirv as i suggest below, we can note that here instead